custom_components/zwave_mqtt/__init__.py

```diff
@@ -90,6 +91,9 @@ def async_node_added(node):
     def async_node_changed(node):
         _LOGGER.debug("[NODE CHANGED] node_id: %s", node.id)
         data_nodes[node.id] = node
+        # notify devices about the node change
+        if node.id not in removed_nodes:
```

Comment:
A bit off topic: When is `removed_nodes` cleared or node_ids removed from `removed_nodes`? It seems we only append to this list but never remove ids.

Should we remove the node id from this list after we have removed the entities of the node? If the user adds the same node back again the node id should no longer be in the removed list, I think.

Reply:
Yeah, but you will never ever get the same node id when you add a new node. But for completeness I'll add it in ;-)

Reply:
Oh. So node ids are not stable for the same device?

Reply:
Node id is just a position in the mesh?

Reply:
The controller will not re-assign a node ID unless you exceed the maximum (232) and then it will wrap back around and re-use any removed ones.